Changes

```diff
@@ -83,6 +83,13 @@ Working version
 - #9282: Make Cconst_symbol have typ_int to fix no-naked-pointers mode.
   (Stephen Dolan, review by Mark Shinwell, Xavier Leroy and Vincent Laviron)
 
+- #9082: The instrumented runtime now records logs in the CTF format.
+  A new API is available in the runtime to collect runtime statistics,
+  replacing the previous instrumented runtime macros.
+  Gc.eventlog_pause and Gc.eventlog_resume were added to allow user to control
+  instrumentation in a running program.
+  (Enguerrand Decorne, Stephen Dolan)
```

Comment:
Please use the same format as the other entries (your name, other contributors, `reviewed by` reviewers) and don't omit any reviewers or contributors...